core/src/main/scala/sttp/tapir/server/interpreter/ServerInterpreter.scala

```diff
@@ -29,11 +29,20 @@ class ServerInterpreter[R, F[_], B, S](
       ses: List[ServerEndpoint[_, _, _, R, F]]
   ): RequestHandler[F, B] = {
     is match {
-      case Nil => (request: ServerRequest) => firstNotNone(request, ses, eisAcc.reverse)
+      case Nil =>
+        new RequestHandler[F, B] {
```

Comment:
maybe it would make sense to add a factory method to `RequestHandler` which would accept a plain Request => F[Response] function?